gammapy/cube/counts.py

```diff
@@ -61,7 +62,7 @@ def fill_map_counts(counts_map, events):
             colnames = [_.upper() for _ in events.table.colnames]
             if axis.name.upper() in colnames:
                 column_name = events.table.colnames[colnames.index(axis.name.upper())]
-                coord_dict.update({axis.name: events.table[column_name].to(axis.unit)})
+                coord_dict.update({axis.name: Quantity(events.table[column_name]).to(axis.unit)})
```

Comment:
I tried calling `.quantity` and still got the error.
Did you ever see the error?

Reply:
No. But didn't @AtreyeeS had a notebook with an `EVENT_ID` `Map`?

The test fails without the `Quantity()`, so it really a regression test.

Reply:
I mentioned it in #1620 already, but that was a weird bug that I didn't manage to fully understand. It only occured if I read from a file, not if I made a Table or Column object from scratch. Up to you whether to let this slide or debug it; it might be a bug in our I/O also, having the events table in a weird state.

Reply:
I suggest we leave #1620 open to figure out where this comes from later.